packages/frontend/src/providers/AppProvider.tsx

```diff
@@ -57,6 +61,19 @@ export const AppProvider: FC = ({ children }) => {
         retry: false,
     });
 
+    useEffect(() => {
+        if (health.data && !isSentryLoaded && health.data.sentry.dsn) {
+            Sentry.init({
+                dsn: health.data.sentry.dsn,
+                release: health.data.sentry.release,
+                environment: health.data.sentry.environment,
+                integrations: [new Integrations.BrowserTracing()],
+                tracesSampleRate: 1.0,
+            });
+            setIsSentryLoaded(true);
+        }
+    }, [isSentryLoaded, setIsSentryLoaded, health.data]);
```

Comment:
Avoid accessing object properties when declaring hook dependencies. 
Instead of `health.data` we should have `health` or we should deconstruct the object beforehand `const {data} = health;` and have `data` as the hook dependency.